.github/workflows/weekly-preview.yml

```diff
@@ -22,15 +22,37 @@ jobs:
     - name: Install setuptools
       run: |
         python -m pip install --user --upgrade setuptools wheel
+    - name: Build OHIF
+      run: |
+        git config --global user.name "CI Builder"
+        git config --global user.email "monai.contact@gmail.com"
+
+        root_dir=$(pwd)
+        ohif_dir=${root_dir}/plugins/ohif
+        
+        pushd $ohif_dir
+        ./build.sh
+        popd
+
+        pushd $ohif_dir/Viewers
+        export YEAR_WEEK=$(date +'%y%U')
+        git add .
+        git commit -m "OHIF Update $YEAR_WEEK"
+
+        git describe --tags --dirty --always --long
+        popd
+
+        git add .
+        git commit -m "OHIF Update $YEAR_WEEK"
+        git status
+        git describe --tags --dirty --always --long
     - name: Build distribution
       run: |
-        export BUILD_OHIF=true
+        export BUILD_OHIF=false
         export HEAD_COMMIT_ID=$(git rev-parse HEAD)
         sed -i 's/name\ =\ monailabel$/name\ =\ monailabel-weekly/g' setup.cfg
         echo "__commit_id__ = \"$HEAD_COMMIT_ID\"" >> monailabel/__init__.py
```

Comment:
this line trying to keep record of `HEAD_COMMIT_ID` of the release, so that when the user report issues we know exactly the version of the source code. But in this case the `HEAD_COMMIT_ID` is pointing at a workaround commit made at line 46 `git commit -m "OHIF Update $YEAR_WEEK"`

Reply:
I added a section to export the HEAD commit id before any additional commits in the scripts